src/avif.c

```diff
@@ -57,8 +57,6 @@ void avifGetPixelFormatInfo(avifPixelFormat format, avifPixelFormatInfo * info)
             break;
 
         case AVIF_PIXEL_FORMAT_YUV400:
-            info->chromaShiftX = 1;
-            info->chromaShiftY = 1;
```

Comment:
As James said, this comes from the AV1 spec (page 32 and page 119). Let's add a comment here to explain this.

The rationale, I heard, is that this allows an AV1 implementation that only supports profile 0 (see seq_profile=0 on page 112 of AV1 spec) to hardcode subsampling_x=1 and subsampling_y=1. (Profile 0 only allows 4:2:0 and monochrome.)